lib/identity_job_log_subscriber.rb

```diff
@@ -179,6 +218,14 @@ def trace_id(job)
   def duplicate_cron_error?(ex)
     ex.is_a?(ActiveRecord::RecordNotUnique) && ex.message.include?('(cron_key, cron_at)')
   end
+
+  def should_error?(ex)
+    return true if ex.nil?
+
+    RiscDeliveryJob.warning_error_classes.none? { |warning_class|
```

Comment:
This should be able to call `job.warning_error_classes` or similar since it's defined on the super class now right?

Reply:
Is it possible to call it from `job`?